dmaws/rds.py

```diff
@@ -295,20 +287,27 @@ def clean_database(self):
                     WHERE (
                         SELECT COUNT(*) FROM supplier_frameworks WHERE supplier_id=suppliers.supplier_id
                     ) = 0
+                    AND supplier_id NOT IN (
+                        SELECT DISTINCT supplier_id FROM services
+                    )
                 ), d1 AS (
                     DELETE FROM contact_information WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
                 ), d2 AS (
                     DELETE FROM users WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
                 )
-             DELETE FROM suppliers WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
+            DELETE FROM suppliers WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
         """)
 
+        # Remove audit events because we don't use them
```

Comment:
After talking about this in person yesterday, we agreed not to preserve any audit events (this is what we're doing now).  We could cherry pick some to help us with given scenarios, but ideally we should just be able to create the audit events we need for our test scenarios. (for example, if we're testing the diff page, it's quite easy to make a simple edit to a service and in that way generate an audit event.)